fix: update readme screenshot

Abdullanabdulaziz opened the pull request:
Updated the screenshot in the README to reflect the latest app interface. The previous screenshot was outdated, and the new one provides a more accurate representation of the application in fullscreen mode. The new screenshot has been framed and styled using Screely for a polished look.

Muhammed-Rahif commented:
Number of things to change:
1. Put the zoom in to 200%
2. Add more content
3. Make the frame like in this example: https://www.reddit.com/media?url=https%3A%2F%2Fpreview.redd.it%2Fnotpad-reinventing-the-ordinary-notepad-experience-with-v0-ypaken2vqi1e1.png%3Fwidth%3D1080%26crop%3Dsmart%26auto%3Dwebp%26s%3Da6f9869ea4d23f63bcccf4eb90b4b54ba2b95a66

Muhammed-Rahif reviewed (changes requested):
To change:
1. Put the zoom in to 200%
2. Add more content
3. Make the frame like in this example: https://www.reddit.com/media?url=https%3A%2F%2Fpreview.redd.it%2Fnotpad-reinventing-the-ordinary-notepad-experience-with-v0-ypaken2vqi1e1.png%3Fwidth%3D1080%26crop%3Dsmart%26auto%3Dwebp%26s%3Da6f9869ea4d23f63bcccf4eb90b4b54ba2b95a66

Abdullanabdulaziz commented:
I’ve updated the screenshot as requested: zoomed to 200%, added more content, and styled the frame to match the example. Let me know if further changes are needed!

Muhammed-Rahif reviewed (changes requested):
- Zoom the Notpad to 200%, not the browser zoom. You can find it inside Notpad application: View > Zoom > Zoom In.
- Turn on line numbers: View > Line Numbers
- Change the editor title from `Untitled.txt` to something appropriate (by double clicking on it).
- Put the screely frame to dark mode:
<img src="https://github.com/user-attachments/assets/faedeb39-22a4-4951-a763-0fcf410fb599" width="300px">

HMSagar2701 commented:
Hi @Muhammed-Rahif, Can I work on this ?

Muhammed-Rahif commented:
> Hi @Muhammed-Rahif, Can I work on this ?
Maybe yeah. @Abdullanabdulaziz was working on this, but no updates after the last change request.

@Abdullanabdulaziz how is it going?

Muhammed-Rahif commented:
Resolved https://github.com/Muhammed-Rahif/Notpad/pull/309
